argus/tests/cloud/windows/test_smoke.py

```diff
@@ -143,3 +144,52 @@ class TestCatchingSpecialize(base.TestBaseArgus):
     def test_traceback_occurred(self):
         instance_traceback = self.introspection.get_cloudbaseinit_traceback()
         self.assertIn('ImportError: No module named mtu', instance_traceback)
+
+
+class TestStaticNetwork(base.TestBaseArgus):
+
+    def test_static_network(self):
+        """Check if the attached NICs were properly configured."""
+        # Get network adapter details within the windows instance.
+        instance_nics = self.introspection.get_nics()
+        # Filter them by DHCP disabled status for static checks.
+        instance_nics = filter(lambda nic: not nic["dhcp"], instance_nics)
```

Comment:
Use a list comprehestion. Plus, this won't work on Python 3 as you are using it right now.